open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/impl/DefaultTileLevelController.java

```diff
@@ -161,12 +170,42 @@ public int getDivisionHoldGeneration()
         return holdGen;
     }
 
+    /**
+     * Gets the current level that is being held at.
+     *
+     * @return the current hold level
+     */
+    public int getCurrentHoldLevel()
+    {
+        return myCurrentHoldLevelProperty.get();
+    }
+
+    /**
+     * Gets the current hold level property.
+     *
+     * @return the current hold level property
+     */
+    public IntegerProperty getCurrentHoldLevelProperty()

```

Comment:
Naming convention on properties deviates from standard Java getters:
```suggestion
    public IntegerProperty currentHoldLevelProperty()
```